### ISSUE TITLE
Question: Using DataTables features that are not in the library

### ISSUE THREAD

**Issue by afbora** [yajra/laravel-datatables#3204]
I will use this library in a project but I am not sure about a few things:

1. Does this library/repository cover all the features in the current version of DataTables?
2. If no, is it possible to add and use features that are not available in the library? In other words, is it flexible or can it be injected?

For example;
As far as I can see, I did not see the [FixedColumns](https://datatables.net/extensions/fixedcolumns/) extension feature in the library. If not, can I use this FixedColumns extension in DataTables with this library?

**Comment by yajra** [comment]
Yes, "most" of the latest features are supported including the premium Editor library. Aside from the server-side Search Builder, can't remember any feature that did not work on my use case. Also, I think there is an issue here it GitHub that has a working solution for server-side search builder.

`FixedColumns` is a client-side feature but is also supported via HTML builder.

Also, the package is open for contributions if ever there are features that are not yet supported.

Thanks!

**Comment by afbora** [comment]
Thank you so much! 💛
